core/__tests__/tasks/profile/checkReady.ts

```diff
@@ -3,7 +3,11 @@ import { api, config, task, specHelper } from "actionhero";
 import { Group, plugin, Profile, ProfileProperty } from "../../../src";
 
 describe("tasks/profile:checkReady", () => {
-  helper.grouparooTestServer({ truncate: true, enableTestPlugin: true });
+  helper.grouparooTestServer({
+    truncate: true,
+    enableTestPlugin: true,
+    disableTestPluginImport: true,
```

Comment:
Ah - looks like you are doing this in more tests so that profiles don't get deleted!

Reply:
Yeah, a lot of tests were relying on the old behavior of keeping the old value instead of clearing it, and setting property values manually.